tests/NBomber.IntegrationTests/ScenarioTests.fs

```diff
@@ -223,6 +225,42 @@ let ``applyScenariosSettings() should override initial settings if the name is m
     test <@ updatedScenarios.[0].WarmUpDuration = warmUp1 @>
     test <@ updatedScenarios.[0].CustomSettings = settings.CustomSettings.Value @>
 
+
+let data : obj[] seq =
+    seq {
+        yield [| Some [|1;0|]; [|1;0|] |]
+        yield [| Some [|1;1|]; [|1;1|] |]
+        yield [| None;         [|0;1|] |]
+    }
+
+[<Theory>]
+[<MemberData("data")>]
+let ``applyScenariosSettings() should reorder steps based on scenario settings``
```

Comment:
can you please also add a real E2E test with step_order_config.json

Reply:
I've added the E2E test, but as for me, it looks ~ugly~ not as clear as I would expect from the test.
But I have no idea how to check step order after running tests. Do you have any idea or current implementation is suitable?